server/src/domain/asset/asset.service.ts

```diff
@@ -272,7 +272,7 @@ export class AssetService {
       zip.addFile(originalPath, filename);
     }
 
-    zip.finalize();
+    await zip.finalize();
```

Comment:
This one breaks downloading files I believe. It needs to not be awaited, so I think we need to add a catch

Reply:
https://github.com/archiverjs/node-archiver/issues/476